backend/models/playlist.js

```diff
@@ -12,13 +12,18 @@ class Playlist {
      * @returns a list of the 5 top playlists using the provided search term
      */
     static async search(term, token) {
-        const url = `https://api.spotify.com/v1/search?q=${term}&type=playlist&limit=5`
+        let words = term.split(" ")
+        if (words.length>4) {
```

Comment:
this logic seems interesting to me. I'm not sure what you are aiming to achieve here. Are you trying to provide the API phrases of 4 words each? Why are you re-joining?